config/jobs/periodic/docker-in-docker/periodic-build-docker.yaml

```diff
@@ -0,0 +1,44 @@
+periodics:
+  - name: periodic-build-docker
```

Comment:
@florencepascual another minor comment,
How about having the job name as `periodic-build-docker-containerd` as your shell scripts build both the packages?

Reply:
Containerd is a dependency of docker, so the focus is mainly on docker. And sometimes, we don't build containerd, in the case where there is no new version of containerd. And we don't build every containerd version, only the dependency of the latest docker version. So I would keep it as it is.